getting-started/templates/systemlink-values.yaml

```diff
@@ -242,7 +242,7 @@ dashboardhost:
   accessControlMirroring:
     ## Enable RBAC mirroring.
     ##
-    enabled: false
+    enabled: true
```

Comment:
Consider just removing this section from the template if it just duplicates the default value.

Reply:
It is redundant. Removed.